api/openapi-spec/swagger.json

```diff
@@ -75018,7 +75018,7 @@
     ],
     "properties": {
      "names": {
-      "description": "Names by which this image is known. e.g. [\"gcr.io/google_containers/hyperkube:v1.0.7\", \"dockerhub.io/google_containers/hyperkube:v1.0.7\"]",
+      "description": "Names by which this image is known. e.g. [\"k8s.gcr.io/hyperkube:v1.0.7\", \"dockerhub.io/google_containers/hyperkube:v1.0.7\"]",
```

Comment:
Does the dockerhub.io/google_containers make sense here? (even if it's still just an example)

Reply:
it never really made sense, IMO